File: app/src/main/java/com/doubean/ford/data/vo/RecommendedGroupResult.java
package com.doubean.ford.data.vo;

import androidx.annotation.NonNull;
import androidx.annotation.Nullable;
import androidx.room.Entity;
import androidx.room.PrimaryKey;

@Entity(tableName = "recommended_groups_results")
public class RecommendedGroupResult {

    @PrimaryKey(autoGenerate = true)
    public Long id;

    @NonNull
    public Integer no;
    @NonNull
    public String groupId;
    @Nullable
    public String postId;

    public RecommendedGroupResult(@NonNull Integer no, @NonNull String groupId, @Nullable String postId) {
        this.no = no;
        this.groupId = groupId;
        this.postId = postId;
    }
}

File: app/src/main/java/com/doubean/ford/data/vo/RecommendedGroupResult.kt
package com.doubean.ford.data.vo

import androidx.room.Entity
import androidx.room.PrimaryKey

@Entity(tableName = "recommended_groups_results")
class RecommendedGroupResult(
    val no: Int, val groupId: String, val postId: String?, @PrimaryKey(autoGenerate = true)
    val id: Long
)